### ISSUE TITLE
 ability to set css.enabledLanguages property in "./vscode/setting.json" or in the ".workspace" file

### ISSUE THREAD

**Issue by snicr** [ecmel/vscode-html-css#286]
Type: **Feature Request / Improvement**

it will be nice to be able to set de "css.enabledLanguages" property in the "./vscode/setting.json" or even in the ".workspace" file.
I'm working with devcontainers and it will be ideal to keep all the related configuration in the code repository, currently this configuration kept in the host machine.

**Comment by ecmel** [comment]
Language ids are registered on extension startup so this may require some refactoring.

**Comment by oestrogen** [comment]
Yes, would be nice to set this setting in the repo and not have to tell the team to set it manually.
